config/application.rb

```diff
@@ -1,6 +1,17 @@
-require File.expand_path('../boot', __FILE__)
+require_relative 'boot'
 
-require 'rails/all'
```

Comment:
As comment below states, we do not want actioncable as of right now (have no need for it). So only require the rails frameworks we need (basically everything but actioncable)